packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -59,8 +60,11 @@ import { HeaderNetworkNotSupported } from '../common/HeaderNetworkNotSupported'
 import { NetworkSelectionContainer } from '../common/NetworkSelectionContainer'
 import { isTestingEnvironment } from '../../util/CommonUtils'
 
-async function addressIsEOA(address: string, provider: JsonRpcProvider) {
-  return (await provider.getCode(address)).length <= 2
+async function addressIsSmartContract(
+  address: string,
+  provider: JsonRpcProvider
+) {
+  return (await provider.getCode(address)).length > 2
```

Comment:
think we can keep the `addressIsEOA` and invert everything used `addressIsSmartContract`

Reply:
dumb q, what does EOA mean?

Reply:
Externally Owned Account aka not smart contract

Reply:
nvm i found out it means `externally owned accounts`

Reply:
tbf I did answered this exact same thing haha